IdentityCore/src/oauth2/MSIDWebviewFactory.m

```diff
@@ -100,11 +100,20 @@ @implementation MSIDWebviewFactory
         return session;
     }
     
+    NSMutableDictionary *headers = [NSMutableDictionary new];
```

Comment:
web view factory is a generic OAuth2 factory. AAD specific things shouldn't leak here. 
Additionally, PkeyAuth headers should only apply to embedded webview and shouldn't impact system webview.

Reply:
Ok, can you provide another suggestion as to how a caller can decide whether or not they want PKeyAuth? I can introduce a boolean, but it felt inelegant.